rdkit/Chem/EState/UnitTestVSA.py

```diff
@@ -12,52 +12,44 @@
 
 """
 from __future__ import print_function
-from rdkit import RDConfig
-import unittest, os
+
+import os
+import unittest
+
 from rdkit import Chem
 from rdkit.Chem.EState import EState_VSA
-import os.path
 
 
-def feq(n1, n2, tol=1e-4):
-  return abs(n1 - n2) <= tol
+class TestCase(unittest.TestCase):
 
+  @staticmethod
```

Comment:
Sorry I was unclear in where I put the comment.
I understand what a staticmethod is. I was wondering why you pulled the code to get the reference data out of the single method in which it was used and put it in a separate method.